labgridhelper/barebox.py

```diff
@@ -17,6 +17,28 @@ def get_commands(command):
                 commands.append(cmd.strip(',').strip(" "))
     return commands
 
+def assert_have_cmds(barebox, expected):
+    """Test if expected commands are available
+    Args:
+        barebox (BareboxDriver): An instance of the BareboxDriver
+        expected: List of strings, each naming an expected command
+    """
+
+    present = set(get_commands(barebox))
+
+    assert set(expected).issubset(present)
```

Comment:
I don't think the asserts belong into labgridhelper, maybe not even into a testsuite.

Reply:
Removed it.